README.md

```diff
@@ -56,14 +56,14 @@ some notion of first-party. In defining this scope, we must balance two goals: t
 small enough to meet the user's privacy expectations, yet large enough to provide the user's desired
 functionality on the site they are interacting with.
 
-Related Website Sets (RWS) is a web platform mechanism, proposed within the context of browser efforts to phase out support for third-party cookies, through which site authors of multi-domain sites may declare relationships between domains such that the browser may understand the relationships and handle cookie access accordingly.
+Related Website Sets (RWS) is a web platform mechanism, proposed within the context of browser efforts to phase out support for third-party cookies, through which site authors of multi-domain sites may declare relationships between domains such that the browser may understand the relationships and handle cross-site data access accordingly.
 
 The core principle of allowing browsers to treat collections of *known related sites* differently from otherwise *unrelated sites* is grounded in ideas that had been previously discussed in the W3C (such as [Affiliated Domains](https://www.w3.org/2017/11/06-webappsec-minutes.html#item12)), the now defunct IETF [DBOUND](https://datatracker.ietf.org/doc/html/draft-sullivan-dbound-problem-statement-02) working group, and previously deployed in some browsers (such as the [Disconnect.me entities list](https://github.com/disconnectme/disconnect-tracking-protection/blob/master/entities.json)).
 
 There are two key components to the proposal:
 
 -   The framework governing how relationships between domains may be declared, and
```

Comment:
Should we also `s/domains/sites/` here, for consistency with the line below?

Reply:
So, this is from the old text - but I agree that in particular the term "multi-domain sites" reads a bit weirdly. WDYT about "multi-site user journeys"?

Reply:
Oh, I was referring to the "framework governing relationships between domains" in line 65, since you updated "cross-domain cookie" to "cross-site cookie" in line 66.

But I agree, "multi-domain sites" does read strangely, and I like your suggested phrasing.

Reply:
Ok, yeah, I can update both.

Reply:
Talked to @cfredric offline - there are a bunch of usages of domain vs. site in this explainer so that it probably doesn't make sense to clean this one up in isolation. I can follow up with a cleanup PR.